test/specs/usage/object-source.spec.js

```diff
@@ -0,0 +1,36 @@
+"use strict";
+
+const { expect } = require("chai");
+const $RefParser = require("../../..");
+const path = require("../../utils/path");
+const setupHttpMocks = require("../../utils/setup-http-mocks");
+const { get } = require("../../../lib/bundle/util/object.js");
+
+describe("Usage", () => {
+  beforeEach(() => {
+    setupHttpMocks({
```

Comment:
hmm, Setupmocks seems like something cool, going to see what it does. This is a note for myself haha.